InfinityCosmos/ForMathlib/AlgebraicTopology/CoherentIso.lean

```diff
@@ -0,0 +1,128 @@
+/-
+Copyright (c) 2024 Johns Hopkins Category Theory Seminar. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johns Hopkins Category Theory Seminar
+-/
+
+import InfinityCosmos.Mathlib.AlgebraicTopology.Nerve
+
+universe u v u' v'
+
+open CategoryTheory Nat
+
+namespace CategoryTheory
+
+/-- This is the free-living isomorphism as a category with objects called
+`zero` and `one`. Perhaps these should have different names?-/
+inductive walkingIso : Type u where
```

Comment:
As this is a type, it should capitalized as `WalkingIso`.

Reply:
Thanks. Fixed!